packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -14,13 +14,34 @@ const double _kCircularProgressIndicatorStrokeWidth = 4.0;
 
 // TODO(hansmuller): implement the support for buffer indicator
 
+/// An abstract material design progress indicator.
```

Comment:
the indicator itself isn't abstract. A base class for material design progress indicators, maybe?